config/spryk/spryks/Spryker/Zed/Business/AddCrudFacade.yml

```diff
@@ -210,3 +210,133 @@ postSpryks:
                       value: "is_active"
                   propertyType:
                       value: "BOOLEAN"
+    - AddZedBusinessFactoryMethod:
+          arguments:
+              factoryMethod:
+                  value: "create{{ domainEntity }}IdentifierBuilder"
+              className:
+                  value: "{{ domainEntity }}IdentifierBuilder"
+              interfaceName:
+                  value: "\\{{ organization }}\\Zed\\{{ module }}\\Business\\{{ domainEntity }}\\Identifier\\{{ domainEntity }}IdentifierInterface"
+              subDirectory:
+                  value: "{{ domainEntity }}/Identifier"
+              returnArray:
```

Comment:
What is that? I guess you wanted to use `output` with its proper value `"\\{{ organization }}\\Zed\\{{ module }}\\Business\\{{ domainEntity }}\\Identifier\\{{ domainEntity }}IdentifierInterface"`.

Same for the other places where you used `returnArray`

Reply:
We need to show that an array is returned in the phpdoc block and in the return type of the method, I did not see that the twig template is ready for this

Reply:
I don't like the solution much, but I can't think of a more compact solution

Reply:
We have filters for it

- typedArrayConvert
- returnValueToDocParameter

You can check the `Common/Method.php.twig` template

Reply:
It seems to be working fine now, didn't get that typedArrayConvert was working so well at first. Thanks for the help, fixed it